Source/Core/Resources/Embed/embed_gen3.py

```diff
@@ -28,8 +28,8 @@ def embed_encounters3():
 
                     string += f"ShadowTemplate({encounter['version']}, {encounter['specie']}, {encounter.get('shiny', 'Shiny::Random')}, {encounter['level']}, {locks}, {len(encounter['locks'])}, {encounter['type']})"
                 else:
-                    string += f"StaticTemplate({encounter['version']}, {encounter['specie']}, {encounter.get('form', 0)}, {encounter.get('shiny', 'Shiny::Random')}, 255, 255, 0, {encounter['level']})"
-                
+                    string += f"StaticTemplate3({encounter['version']}, {encounter['specie']}, {encounter.get('form', 0)}, {encounter.get('shiny', 'Shiny::Random')}, 255, 255, 0, {encounter['level']}, {str(encounter['buggedRoamer']).lower()})"
```

Comment:
Don't need this here, just have it handled in the constructor of StaticTemplate3

Reply:
Changed